packages/openapi-react-query/README.md

```diff
@@ -0,0 +1,67 @@
+# openapi-react-query
+
+openapi-react-query is a type-safe tiny wrapper (1 kb) around [@tanstack/react-query](https://tanstack.com/query/latest/docs/framework/react/overview) to work with OpenAPI schema.
+
+It works by using [openapi-fetch](../openapi-fetch) and [openapi-typescript](../openapi-typescript) so you get all the following features:
+
+- ✅ No typos in URLs or params.
+- ✅ All parameters, request bodies, and responses are type-checked and 100% match your schema
+- ✅ No manual typing of your API
+- ✅ Eliminates `any` types that hide bugs
+- ✅ Also eliminates `as` type overrides that can also hide bugs
+
+## Setup
+
+Install this library along with [openapi-fetch](../openapi-fetch) and [openapi-typescript](../openapi-typescript):
+
+```bash
+npm i openapi-react-query openapi-fetch
+npm i -D openapi-typescript typescript
+```
+
+Next, generate TypeScript types from your OpenAPI schema using openapi-typescript:
+
+```bash
+npx openapi-typescript ./path/to/api/v1.yaml -o ./src/lib/api/v1.d.ts
+```
+
+> ⚠️ Be sure to <a href="https://redocly.com/docs/cli/commands/lint/" target="_blank" rel="noopener noreferrer">validate your schemas</a>! openapi-typescript will err on invalid schemas.
```

Comment:
```suggestion
::: warning

⚠️ Be sure to <a href="https://redocly.com/docs/cli/commands/lint/" target="_blank" rel="noopener noreferrer">validate your schemas</a>! openapi-typescript will err on invalid schemas.

:::
```

The `>` creates a blockquote which is the wrong screenreader element for this (if other instances of `>` exist in the docs that aren’t quoting someone they should be updated; not as part of this PR, just in general)

Reply:
It does not seem to be a valid markdown syntax. Are you sure you are not mistaking with the vitepress doc?

Reply:
I am! Apologies. This is fine for READMEs!